tests/test_feed_storage_module/test_hay.py

```diff
@@ -165,14 +165,17 @@ def test_calculate_additional_dry_matter_loss(
     expected: float,
 ) -> None:
     """Tests _calculate_additional_dry_matter_loss in Hay."""
+    im = InputManager()
+    mocker.patch.object(im, "get_data", return_value=size)
+    hay = Hay()
```

Comment:
Instead of changing this test and making it inconsistent with the rest of Feed Storage test suite, I suggest just setting the bale size directly.
```suggestion
    hay.bale_size = size
```

Reply:
Curious about this as well.